packages/genesys-spark-components/src/components/stable/gux-modal/example.html

```diff
@@ -45,24 +41,17 @@ <h2>No title</h2>
   >
   <gux-modal id="example2" size="small">
     <div slot="content">This contains the modal content.</div>
-    <div slot="start-align-buttons">
-      <gux-button-slot accent="primary">
-        <button type="button">Save</button>
-      </gux-button-slot>
-      <gux-button-slot>
```

Comment:
I think you should maybe keep these examples here and include a separate cta group example since they will still be relevant.

Reply:
These slots are to be deprecated in V5 so we don't want anybody new using them in the meantime.

Reply:
Here's the V5 ticket:
https://inindca.atlassian.net/browse/COMUI-3138